### PR TITLE
Allow rendering of dates on X axis in multiChart

### PR DESCRIPTION
**Pull request by syndesis**
This change allows the rendering of dates in multiChart as described in issue #1228.  It adds a config option called xAxisDates.

### CODE REVIEW

File: src/models/multiChart.js [hunk 1]
```diff
@@ -90,8 +91,8 @@ nv.models.multiChart = function() {
                     })
                 });
 
-            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return d.x } ))
-                .range([0, availableWidth]);
+            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) {
+                return xAxisDates ? new Date(d.x) : d.x } )).range([0, availableWidth]);
```
Comment (liquidpele) [inline src/models/multiChart.js]: Hmm... Seems like this should be using chart.x()(d) to get the value... Then you could just override the x getter option to return a date....

Comment (syndesis) [inline src/models/multiChart.js]: That does seem like a better idea, yes.

**Comment by syndesis** [comment]
Allow rending of dates (or anything) on X axis in multiChart.  Using new fix suggested by @liquidpele in #1228

File: src/models/multiChart.js [hunk 2]
```diff
@@ -90,7 +90,7 @@ nv.models.multiChart = function() {
                     })
                 });
 
-            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return d.x } ))
+            x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return chart.x()(d) } ))
```
Comment (liquidpele) [inline src/models/multiChart.js]: Oh, it looks like right above this it uses the getX function (which is what chart.x() actually returns)... so should probably just use that for consistency...     sorry!   

If you change that (and please verify it actually works for your use case) then I'll merge it in :)

**Comment by syndesis** [comment]
Updated to use getX.. still works in my use case.

**Comment by syndesis** [comment]
Oops, just noticed this included some unrelated changes I was working on.  Will fix that.

**Comment by syndesis** [comment]
Sorry, pushed an unrelated change.  Will open a new issue for that.

**Comment by liquidpele** [comment]
awesome, thanks for contributing!

**Pull request merged**